rllib/tests/test_ray_client.py

```diff
@@ -36,45 +37,24 @@ def test_cartpole_lstm(self):
         with ray_start_client_server():
             assert ray.util.client.ray.is_connected()
 
-            config = dict(
-                {
-                    "num_sgd_iter": 5,
-                    "model": {
-                        "vf_share_layers": True,
-                    },
-                    "vf_loss_coeff": 0.0001,
-                },
-                **{
-                    "env": StatelessCartPole,
-                    # Use GPUs iff `RLLIB_NUM_GPUS` env var set to > 0.
-                    "num_gpus": int(os.environ.get("RLLIB_NUM_GPUS", "0")),
-                    "model": {
-                        "use_lstm": True,
-                        "lstm_cell_size": 256,
-                        "lstm_use_prev_action": None,
-                        "lstm_use_prev_reward": None,
-                    },
-                    "framework": "tf",
-                    # Run with tracing enabled for tfe/tf2?
-                    "eager_tracing": None,
-                })
+            config = {
+                "env": StatelessCartPole,
+            }
 
             stop = {
-                "training_iteration": 200,
-                "timesteps_total": 100000,
-                "episode_reward_mean": 150.0,
```

Comment:
In general, how does one tell if for a particular test it's ok to relax the stop criteria? I have that question multiple times looking at rllib's test set up.
Going from 200 --> 3 and dropping reward_mean is a huge reduction.

Reply:
Great question! It's just that this particular test is taking a long time and the same learning capability is already tested elsewhere, so I scrapped this and just make sure it runs.